-------------------------------------------------------------------------------
Test set: redis.clients.jedis.tests.JedisClusterTest
-------------------------------------------------------------------------------
Tests run: 53, Failures: 0, Errors: 53, Skipped: 0, Time elapsed: 0.034 sec <<< FAILURE! - in redis.clients.jedis.tests.JedisClusterTest
testClusterCountKeysInSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testClusterCountKeysInSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testRedisClusterMaxRedirections(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testRedisClusterMaxRedirections(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testClusterKeySlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testClusterKeySlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testSetClientName(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testSetClientName(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testRedisHashtag(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testRedisHashtag(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testAskResponse(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testAskResponse(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testIfPoolConfigAppliesToClusterPools(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testIfPoolConfigAppliesToClusterPools(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testReadonly(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testReadonly(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.002 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testDiscoverNodesAutomatically(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testDiscoverNodesAutomatically(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testMigrateToNewNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testMigrateToNewNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testMovedExceptionParameters(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testMovedExceptionParameters(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testThrowMovedException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testThrowMovedException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testCloseable(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testCloseable(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testLocalhostNodeNotAddedWhen127Present(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testLocalhostNodeNotAddedWhen127Present(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testClusterFlushSlots(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testClusterFlushSlots(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testClusterForgetNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testClusterForgetNode(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testInvalidStartNodeNotAdded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testInvalidStartNodeNotAdded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testCalculateConnectionPerSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testCalculateConnectionPerSlot(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testMigrate(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testMigrate(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testReturnConnectionOnJedisConnectionException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testReturnConnectionOnJedisConnectionException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testThrowAskException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testThrowAskException(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testStableSlotWhenMigratingNodeOrImportingNodeIsNotSpecified(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testReturnConnectionOnRedirection(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testReturnConnectionOnRedirection(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testJedisClusterTimeout(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testJedisClusterTimeout(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testRecalculateSlotsWhenMoved(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testRecalculateSlotsWhenMoved(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

testJedisClusterRunsWithMultithreaded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.setUp(JedisClusterTest.java:68)

testJedisClusterRunsWithMultithreaded(redis.clients.jedis.tests.JedisClusterTest)  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
	at redis.clients.jedis.tests.JedisClusterTest.tearDown(JedisClusterTest.java:128)

redis.clients.jedis.tests.JedisClusterTest  Time elapsed: 0.001 sec  <<< ERROR!
redis.clients.jedis.exceptions.JedisConnectionException: Failed connecting to host localhost:7379
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at redis.clients.jedis.tests.JedisClusterTest.cleanUp(JedisClusterTest.java:116)

